engine/runtime-compiler/src/main/java/org/enso/compiler/pass/analyse/PassPersistance.java

```diff
@@ -128,7 +131,9 @@ public PersistAliasAnalysisGraphScope() {
     @SuppressWarnings("unchecked")
     protected Graph.Scope readObject(Input in) throws IOException {
       var childScopes = in.readInline(scala.collection.immutable.List.class);
-      var occurrences = (scala.collection.immutable.Set) in.readObject();
+      var occurrencesValues = (scala.collection.immutable.Set<Graph.Occurrence>) in.readObject();
+      var ev = ($less$colon$less<Graph.Occurrence, Tuple2<Object, Graph.Occurrence>>) null;
```

Comment:
What a nicely typed `null`! Is it needed at all to have a typed `null` when there is `@SuppressWarnings("unchecked")`?

Reply:
How is this even possible to be a subtype of a tuple?
```scala
Graph.Occurrence <:< (Object, GraphOccurrence)
```

Reply:
This is what I get for trusting IDE